heudiconv/parser.py

```diff
@@ -7,8 +7,9 @@
 
 from collections import defaultdict
 
-import tarfile
-from tempfile import mkdtemp
+import shutil
+import typing
```

Comment:
I think frequent `typing.` makes code too cluttered. I am ok with imports of the constructs we use as we already do elsewhere

```
heudiconv/dicoms.py:from typing import List, Optional
```

Reply:
note -- ATM we do not check typing information at all. Filed https://github.com/nipy/heudiconv/issues/653

Reply:
Sounds good. Looking forward to newer versions that allow the use of builtins like `list` or `|` :)